openedx/features/smart_referral/tasks.py

```diff
@@ -3,6 +3,8 @@
 
 from common.lib.mandrill_client.client import MandrillClient
 
+from openedx.features.smart_referral.models import SmartReferral
```

Comment:
Please run isort or manually remove line 5.

Reply:
I did run isort for sorting imports. If the line break is concerning than we are following a practice to add line break between python, django and project imports. This practice is following in many occurrences  within our project.

Reply:
I agree, but aren't these both project imports? And when I run isort in this file, inside lms-shell, it removes this line break.

Reply:
@muak50